ember-cli/2018-07/july-26.md

```diff
@@ -0,0 +1,7 @@
+# ember-cli Meeting Notes - 2018-07-26
+
+## Attendees
+
+## Agenda
```

Comment:
Enter items here...

Reply:
* broccoli 2
* ember-template-lint
* jQuery via @ember/optional-features
* travis stages
* eslint during build

Reply:
https://github.com/ember-cli/ember-cli/pull/7941

Reply:
also, discuss dep collisions during the build time

Reply:
https://github.com/ember-cli/ember-cli/pull/7935